### DISCUSSION TITLE
有没有更简单的自定义快捷键的方法？

### DISCUSSION THREAD

**Discussion by dreamwar2005** [the1812/Bilibili-Evolved#2223] [综合讨论 / General]
我按照教程用浏览器的控制台输入代码修改默认快捷键，总是有失效的情况。可以的话能否设置一个自定义快捷键修改的界面？

如果工作量大的话可以把几个常用的快捷键提供自定义设置，如倍速调整，弹幕开关，宽屏全屏等。

PS：修改快捷键的时候用小键盘的数字是怎么修改？我用“num0”或“number0”都没有反应。

**Comment by the1812** [comment]
> 我按照教程用浏览器的控制台输入代码修改默认快捷键，总是有失效的情况。可以的话能否设置一个自定义快捷键修改的界面？

这个只有 v2.0 中支持
<img src="https://user-images.githubusercontent.com/26504152/131054150-033e174c-ce0d-4d72-a0c4-eb8ffe009232.png" width="400">

> 修改快捷键的时候用小键盘的数字是怎么修改？我用“num0”或“number0”都没有反应。

试试 numpad 0
